.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: ruby
+rvm:
+  - 2.4.0
+
+script: 
+  - bundle exec bin/rake db:create 
+  - bundle exec bin/rake db:migrate 
+  - bundle exec rspec spec
+
+deploy:
+  provider: heroku
+  app: ace-book
+  api_key:
+    secure: "Zhq+bAklzTvNgYeVKtA5mi3GmTYqEzCWCGclgOuK20D23Yi1iPkbcH09ricLABlk1Y7qFB8jGtbplrPbGrQQdhUW9sQ9OGLRlA/vJnMhU4IVK/ZM1EJCFZPZPv5Zbg4s8L2r0Gz7uIEOX05TYFcREZ5iice4Mq89rX/SgOVPswh3KCFq0BfxaLq111WNAJtApH9amhuJbOUnRB7zMu38kpJcUqMsCFK3jeYBMbp4GxTyg+KK9MtiHS1tOrtXttx5EGaRyzBVi3JgFBo7gTaL3d03FJ6S5/3RFtDol3cUMSIfUVJkFlnzoedmyIgRq+PeDFIUZ17U6ZW8H09Rz9DjXBrpbzIGTEH92Xij7Fqd/BB2PE0ygSFwMcAxGvJHE5WJqNe6T5BhkATxGPXlYwRWMg3szZJdvlsu68xn6xAPdm2xH1DvdL1/ptcWltQFThjp3iy494sODbUInnjHPrj6zr1GsYlWnHigjcqi0IoJFjZRe55gM31cAadFYSBtm9wVoZHzyEje4MmjUwSUZMlaUVkv28eN5rdt8dbZYznu6IPUEY1nTFFipyiDq9WwidyziS4D6Flm5pbcNQBHyg+YwC1fzu3mcMSBnJtzqA60+IYao6iI+uI1eMX3+gSNdUVfEug8O6nuBxD5nCyojMlqmczai/s5oRJeTMT3PvITOV4="
```

Comment:
Woah this should not be in git. What is that the api key for? Heroku? Are you 100% sure you need this in the yml file? I would suggest immediately changing this key in heroku, as it is now compromised.